### PR TITLE
hough_circle radius accepts scalars and lists

### PR DESCRIPTION
**Pull request by juliusbierk**

### CODE REVIEW

**Comment by emmanuelle** [comment]
Thank you! Can you please also update the Parameters section in the docstring (radius should be "scalar or sequence of scalars", although I'm not sure whether radius has to be an int or not - does it make sense to have a non-int radius? Can you check it please?).

Also, please add a test where you give a scalar argument in ``test_hough_transform.py``

File: skimage/transform/tests/test_hough_transform.py [hunk 1]
```diff
@@ -140,8 +140,9 @@ def test_hough_circle():
     y, x = circle_perimeter(y_0, x_0, radius)
     img[x, y] = 1
 
+    out = tf.hough_circle(img, radius)
+    out = tf.hough_circle(img, [radius])
```
Comment (ahojnnes) [inline skimage/transform/tests/test_hough_transform.py]: We should also test, whether these actually produce the same result.

Comment (juliusbierk) [inline skimage/transform/tests/test_hough_transform.py]: Good point, added.

**Comment by juliusbierk** [comment]
Commit fixes #1682

File: skimage/transform/tests/test_hough_transform.py [hunk 2]
```diff
@@ -140,8 +140,11 @@ def test_hough_circle():
     y, x = circle_perimeter(y_0, x_0, radius)
     img[x, y] = 1
 
+    out1 = tf.hough_circle(img, radius)
+    out2 = tf.hough_circle(img, [radius])
+    assert_equal(out1,out2)
```
Comment (emmanuelle) [inline skimage/transform/tests/test_hough_transform.py]: pep8: missing a whitespace after comma (sorry for the horrible nitpicking :-))

File: skimage/transform/tests/test_hough_transform.py [hunk 3]
```diff
@@ -140,8 +140,11 @@ def test_hough_circle():
...
     out = tf.hough_circle(img, np.array([radius], dtype=np.intp))
-
+    assert_equal(out,out1)
```
Comment (emmanuelle) [inline skimage/transform/tests/test_hough_transform.py]: same here

Comment (juliusbierk) [inline skimage/transform/tests/test_hough_transform.py]: ah, sorry. I need to have a linter running on every save!

**Comment by emmanuelle** [comment]
Looks good to me, thanks! Waiting for Travis now...

**Comment by ahojnnes** [comment]
LGTM, thanks very much!

**Comment by stefanv** [comment]
Closes #1682.

**Pull request merged**